client/grpc/client.go

```diff
@@ -29,22 +30,22 @@ type grpcClient struct {
 }
 
 // New creates a drand client backed by a GRPC connection.
-func New(address, certPath string, insecure bool, chainHash []byte) (client.Client, error) {
+func New(address, certPath string, notsecure bool, chainHash []byte) (client.Client, error) {
```

Comment:
insecure is better english but it might be better to expose two funcs `NewInsecureClient` and `NewClient` to avoid boolean params

Reply:
insecure was clashing with the grpc `insecure` package and the variable was shadowing it :P 
I guess I can rename the grpc package instead if you prefer.